src/main/java/mate/academy/bookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -15,15 +13,21 @@
 
 @RestControllerAdvice
 public class CustomGlobalExceptionHandler {
+    @ExceptionHandler(Exception.class)
+    public ErrorResponse handleException(Exception ex) {
+        return ResponseHandler.getErrorResponse(
+                ex.getMessage(), HttpStatus.BAD_REQUEST);
```

Comment:
I wouldn't treat ANY exception taht we have in our app as a bad request by default, rather a `500 - internal server error`